docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+#
+# 
+# Build
+# docker-compose build
+#
+# Run
+# docker-compose up -d 
+#
+version: "3.3"
+ 
+services:
+  revgeod:
+    build:
+      context: ./
+    container_name: revgeod
+    environment:
+      #- OPENCAGE_APIKEY=${OPENCAGE_APIKEY}     # set this in .env
```

Comment:
"set this in .env" ? Here or there? If either/or: which is better/easier/faster/cheaper? ;)

Reply:
Since .env is in .gitignore the preferred way is to use .env to avoid accidently checking in your API key(s)